Client/Frontend/Components/Pages/CheckIn/Update/EmotionCheckInUpdate.razor

```diff
@@ -0,0 +1,46 @@
+﻿@page "/CheckIn/EmotionCheckInUpdate/{id:int}/{emotion}"
+@using API.DTO
+@using Frontend.Services
+@using Frontend.Services.Interfaces
+@inject NavigationManager NavigationManager
+@inject IEmotionCheckInService EmotionCheckInService
+
+<h3>Update Emotion Check-In</h3>
+
+@if (emotionCheckIn != null)
+{
+  <form method="post" @onsubmit="Submit">
+    <div class="form-group">
+      <label for="description">Description</label>
+      <input type="text" class="form-control" id="description" @bind="emotionCheckIn.Id" placeholder="Enter Description">
```

Comment:
<img width="1820" alt="image" src="https://github.com/user-attachments/assets/6215d823-d5c7-4f40-96c7-695546566a37">
this is how your website looks and this is how it is in figma
<img width="603" alt="image" src="https://github.com/user-attachments/assets/3a061493-a7f0-4d32-a8ea-8a57f19b42fd">
it doesn't really display what emotion you chose :/

Reply:
its nothing like the figma one